website/docs/os.getSystemTags.md

```diff
@@ -0,0 +1,19 @@
+Returns the list of tags corresponding to a system.
+
+There are tags specific to each operating system (see [system()](system.md) for a complete list of identifiers.),
+and meta tags like `posix`, `darwin`, `desktop` and `mobile` tags.
+
+### Examples ###
+```lua
+print("iOS system tags: " .. os.getSystemTags("ios"))
+-- { "ios", "darwin", "posix", "mobile" },
+```
+
+### See Also ###
+[os.istarget](os.istarget.md)
+[os.target](os.target.md)
+
+### Availability ###
+
+Premake 5.0.0 beta 3 or later.
```

Comment:
This function has been around since alpha 12. This is the commit that introduced it, you can see the tags listed at the top: 70e00883